src/InstrumentationEngine.NuGet.Headers/InstrumentationEngine.Nuget.Headers.csproj

```diff
@@ -2,13 +2,21 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved.
      Licensed under the MIT License. -->
 <Project>
+  <PropertyGroup Condition="'$(Platform)'=='ARM64'">
+    <PackageId>Microsoft.InstrumentationEngine.Headers.ARM64</PackageId>
```

Comment:
Are we ever going to publish this package to nuget.org? Is this just a temporary internal package until everything is validated E2E?

Reply:
Sort of, we didn't want to modify the currently public headers package due to the blocking internal ARM64 sdk requirement. This header is in case the clients in VS need a separate headers package but long term we hope to integrate it with the currently public headers package once everything gets plumbed through.